Improved RTT section (#60)
Haivision/srt-rfc: draft-sharabayko-mops-srt.md

In the following formula, RTT is the current value that the receiver
maintains and rtt is the recent value that was just calculated from an ACK/ACKACK pair:

~~~
RTT = RTT * 0.875 + rtt * 0.125
~~~

RTT variance RTTVar is obtained as follows:

~~~
RTTVar = RTTVar * 0.75 + abs(RTT - rtt) * 0.25
~~~

where abs() means an absolute value.

Both RTT and RTTVar are measured in microseconds. The initial value of RTT is 100 milliseconds, RTTVar is 50 milliseconds.

The smoothed RTT calculated by the receiver as well as the RTT variance
RTTVar are sent with the next full acknowledgement packet (see section {{ctrl-pkt-ack}}).
Note that the first ACK in an SRT session might contain an initial RTT
value of 100 milliseconds, because the early calculations may not be precise.

The sender always gets the RTT from the receiver. It does not have an analog to the
ACK/ACKACK mechanism, i.e. it can not send a message that guarantees an immediate return
without processing. Upon an ACK reception, the SRT sender updates its own RTT and RTTVar
values using the same formulas as above, in which case rtt is the most recent value it receives,
e.g., carried by an incoming ACK.

> Note that SRT socket can both send and receive. RTT and RTTVar are not just sender specific variables.
> They are shared by both the sender and the receiver (of the same socket).
> When an SRT socket receives data, it updates its local RTT and RTTVar, which can be used for its own sender as well.

Reviewer: ```suggestion
Note that SRT socket can both send and receive data packets. RTT and RTTVar are updated by the socket both as a sender algorithm (via ACK packet), and as a receiver algorithm (via ACK-ACKACK pair).
When an SRT socket receives data, it updates its local RTT and RTTVar, which can be used for its own sender as well.
```

sent out and a corresponding ACKACK packet is received back by the
SRT receiver.

> An ACK sent by the receiver triggers sending an ACKACK by the sender
> with minimal processing delay. The receiver is expected to receive the ACKACK response
> roughly one RTT after the corresponding ACK was sent.

Reviewer 2: ```suggestion
An ACK sent by the receiver triggers an ACKACK from the sender with 
minimal processing delay. The ACKACK response is expected to arrive 
at the receiver roughly one RTT after the corresponding ACK was sent.
```

ACK/ACKACK mechanism, i.e. it can not send a message that guarantees an immediate return
without processing. Upon an ACK reception, the SRT sender updates its own RTT and RTTVar
values using the same formulas as above, in which case rtt is the most recent value it receives,
i.e., carried by an incoming ACK.

> Note that SRT socket can both send and receive data packets. RTT and RTTVar are updated by the socket both as a sender algorithm (via ACK packet), and as a receiver algorithm (via ACK-ACKACK pair).

Reviewer 2: ```suggestion
Note that an SRT socket can both send and receive data packets. RTT and RTTVar are updated by the socket based on algorithms for the sender (using ACK packets) and for the receiver (using ACK-ACKACK pairs).
```
